etna/articles/models.py

```diff
@@ -512,6 +540,48 @@ def latest_items(
         )[:3]
 
 
+class PageGalleryImage(Orderable):
+    page = ParentalKey(Page, on_delete=models.CASCADE, related_name="gallery_images")
+    image = models.ForeignKey(
+        get_image_model_string(), on_delete=models.SET_NULL, null=True, related_name="+"
+    )
+    alt_text = models.CharField(
+        verbose_name=_("alternative text"),
+        max_length=100,
+        help_text=mark_safe(
+            'Alternative (alt) text describes images when they fail to load, and is read aloud by assistive technologies. Use a maximum of 100 characters to describe your image. <a href="https://html.spec.whatwg.org/multipage/images.html#alt" target="_blank">Check the guidance for tips on writing alt text</a>.'
+        ),
+    )
+    caption = RichTextField(
+        features=["bold", "italic", "link"],
+        help_text="An optional caption, which will be displayed directly below the image. This could be used for image sources or for other useful metadata.",
+        blank=True,
+    )
+
+    class Meta(Orderable.Meta):
+        verbose_name = _("gallery image")
+        verbose_name_plural = _("gallery images")
+
+    panels = [
+        FieldPanel("image"),
+        FieldPanel("alt_text"),
+        FieldPanel("caption"),
+    ]
+
+
+class GallerySerializer(serializers.ModelSerializer):
+    image = HighlightImageSerializer(rendition_size="max-1024x1024")
```

Comment:
Until we properly design the gallery component, these images need a specific PNG rendition with a `max-900x900` size.

Reply:
We're using 1024x1024 at the minute in the Highlight Gallery
```
{% image first_gallery_item.image max-1024x1024 format-webp as image_webp %}
{% image first_gallery_item.image max-1024x1024 as image_png %}
{% image first_gallery_item.image max-1024x1024 as base_img %}
```

I'll change how this gets the images to allow for PNG and other image types :)

Reply:
It's fine - we'll stick with 1024 x 1024 for now until we can rebuild the component 👍🏼